Jenkinsfile

```diff
@@ -263,6 +263,8 @@ def buildProject(targetPlatform, forceLocalDeployment = false, tychoVersion = "4
       GOALS='clean deploy'
       if [ "${FORCE_LOCAL_DEPLOYMENT}" == "true" ] || [ "${BRANCH_NAME}" != "master" ] && [ "${RELEASE_TYPE}" == "Integration" ] && [ "${FORCE_PUBLISH}" != "true" ]; then
         GOALS="${GOALS} -DaltDeploymentRepository=snapshot-repo::default::file:./my-local-snapshots-dir"
+      else
+        GOALS="${GOALS} -Psonatype-oss-release"
```

Comment:
@LorenzoBettini is that what you thought of?

Reply:
and wont this call the old deploy plugin in the else then too?

Reply:
no it does not work.
the old deploy plugin is called and crashes

Reply:
What's the error of the failure/crash?

Reply:
aahhhh no it does not.